.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,39 @@
+---
+name: bump-version
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - elevate-cpanel
+  workflow_dispatch:
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: true  # Fetch Hugo themes (true OR recursive)
+        fetch-depth: 0    # Fetch all history for .GitInfo and .Lastmod
+
+    - name: bump version number
+      run: |
+        set -x
+        VERSION=$(cat ./version)
+        let ++VERSION
+        echo -n $VERSION > ./version
+        sed -i -re "/^#<<V/,+1 s/VERSION => [0-9]*;/VERSION => ${VERSION};/" ./elevate-cpanel
```

Comment:
This needs to go in a Makefile (make release) not in a workflow.

Reply:
I'm not ok with a version bump except when we do a release. Let's make this a Makefile

Reply:
Version bump is now accomplished via Makefile:
```
[root@sandbox elevate]# make -n release
echo 1 > version
sed -i -re "/^#<<V/,+1 s/VERSION => [0-9]*;/VERSION => 1;/" elevate-cpanel
/usr/local/cpanel/3rdparty/bin/git commit -m "Release version 1" -- version elevate-cpanel
/usr/local/cpanel/3rdparty/bin/git tag -f release
/usr/local/cpanel/3rdparty/bin/git push origin
/usr/local/cpanel/3rdparty/bin/git push --force origin tag release
```